content/2019/07/01/smashing-variables.rst

```diff
@@ -0,0 +1,20 @@
+public: yes
+author: miriam
+canonical: https://www.smashingmagazine.com/2019/07/css-custom-properties-cascade/
+elsewhere: Smashing Magazine
+tags: [Elsewhere, CSS, Custom Properties, Smashing Magazine]
```

Comment:
Are we using single quotes around multi-word tags for consistency? https://deploy-preview-359--oddsite.netlify.com/styleguide/#tags